admin-su/pom.xml

```diff
@@ -5,7 +5,7 @@
     <artifactId>base</artifactId>
     <groupId>org.sakaiproject</groupId>
     <version>19-SNAPSHOT</version>
-    <relativePath>../../pom.xml</relativePath>
+    <relativePath>../pom.xml</relativePath>
```

Comment:
Should this be ../master/pom.xml instead?

Reply:
The <artifactId> of `./pom.xml` its called `base` while the <artifactId> of `./master/pom.xml` its called `master`.
So, my interpetation of what the original developer was trying to referenciate its to point `./pom.xml`.

Reply:
The parent for this tool should be master:
```
    <parent>
        <groupId>org.sakaiproject</groupId>
        <artifactId>master</artifactId>
        <version>19-SNAPSHOT</version>
        <relativePath>../master/pom.xml</relativePath>
    </parent>
```

Reply:
Changed.
NVM, I think that `<version>` and `<relativePath>` can be deleted. (Not sure)